common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/ElasticClientBuilder.scala

```diff
@@ -31,9 +32,9 @@ object ElasticClientBuilder {
       .setHttpClientConfigCallback(new ElasticCredentials(username, password))
       // Needed for the snapshot_generator.
       // TODO Make this a flag
-      .setMaxRetryTimeoutMillis(2000)
+//      .setMaxRetryTimeoutMillis(2000)
```

Comment:
Is this not needed anymore? Maybe just delete this rather than commenting out if so?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/7.x/breaking-changes-7.0.html#maxretrytimeout-removed

Thanks for spotting.